docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -65,7 +65,7 @@
     "    import cirq\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        sampler=cirq.Simulator(),  # Any simulator or a ProcessorSampler.
</pre><p>Can we maybe use a simulator with noise for this example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>